backend/python/app/services/implementations/story_service.py

```diff
@@ -561,6 +565,9 @@ def _get_num_approved_lines(self, translation_contents):
 
         return count
 
+    def _get_num_content_lines(self, translation_contents):
```

Comment:
i think this function is not entirely necessary, feel free to just
`response["num_content_lines"] = len(response["translation_contents"])`